src/Nethermind/Nethermind.Synchronization/ParallelSync/SyncMode.cs

```diff
@@ -70,5 +70,33 @@ public enum SyncMode
     public static class SyncModeExtensions
     {
         public static bool NotSyncing(this SyncMode syncMode) => syncMode == SyncMode.WaitingForBlock || syncMode == SyncMode.Disconnected;
+
+        public static bool IsSyncingBodies(this SyncMode syncMode) =>
```

Comment:
What about SyncMode.Disconnected? :)

Reply:
Should we consider Disconnected as syncing?

Reply:
Above `NotSyncing` - says directly that we are not syncing :)
Lets keep the warning then.